easistrain/EDD/calibrationEDD.py

```diff
@@ -144,21 +134,22 @@ def guessParameters(xData, yData, counterOfBoxes, nbPeaksInBoxes):
     return firstGuess, peaksGuess
 
 
-### calibEdd is the main function
 def calibEdd(
-    fileRead,
-    fileSave,
-    sample,
-    dataset,
-    scanNumberHorizontalDetector,
-    scanNumberVerticalDetector,
-    nameHorizontalDetector,
-    nameVerticalDetector,
-    numberOfBoxes,
-    nbPeaksInBoxes,
-    rangeFit,
-    sourceCalibrantFile,
+    fileRead: str,
+    fileSave: str,
+    sample: str,
+    dataset: Union[str, int],
+    scanNumberHorizontalDetector: Union[str, int],
+    scanNumberVerticalDetector: Union[str, int],
+    nameHorizontalDetector: str,
+    nameVerticalDetector: str,
+    numberOfBoxes: int,
+    nbPeaksInBoxes: Sequence[int],
+    rangeFit: Sequence[int],
+    sourceCalibrantFile: str,
```

Comment:
I added some type indications. These do not change anything to the code execution but are a nice addition for the developer to know what kind of arguments are expected.